lib/gyro/infos.rb

```diff
@@ -0,0 +1,29 @@
+# Gyro Infos
+#
+module Gyro	
+	module Infos
+		TEMPLATE_FILE = "./lib/templates"
+		def self.listTemplates
+			entries = Dir[Infos::TEMPLATE_FILE+"/*"] 
```

Comment:
That function I'm talking about in the comment above returns a `Pathname` (and not a string) pointing to the templates directory. `Pathname` is a ruby class dedicated to manipulate files and folders. Look at its documentation I'm pretty sure there's a method to list all the children of a directory. So you could do: 

```ruby
list = Gyro::Liquid::Generator.templates_dir.children
list.filter { |e| e.directory? }.each do |dir|
  puts " - #{dir}"
end
```